openqa-label-known-issues-and-investigate-hook

```diff
@@ -46,4 +55,18 @@ hook() {
     fi
 }
 
+usage() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+"hook script" intended to be called by openQA instances taking a job ID as
+parameter and forwarding a complete job URL to "openqa-label-known-issues" on
+stdin and all left unknowns to "openqa-investigate"
```

Comment:
What do 'left unknowns' refer to? Is this dependent on the output of openqa-label-known-issues that tells this script if a job is unknown (whatever that means)?